packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapterInit.ts

```diff
@@ -120,9 +130,20 @@ export const initialize = async ({
     addEventHandler(environment.handleTestEvent.bind(environment));
   }
 
+  // @ts-expect-error
+  const runtimeGlobals: JestGlobals = {expect, ...globals};
+  // TODO: `jest-circus` might be newer than `jest-runtime` - remove `?.` for Jest 27
+  setGlobalsForRuntime?.(runtimeGlobals);
+
+  // TODO: `jest-circus` might be newer than `jest-config` - remove `??` for Jest 27
+  if (config.injectGlobals ?? true) {
+    Object.assign(environment.global, runtimeGlobals);
+  }
+
   await dispatch({
     name: 'setup',
     parentProcess,
+    runtimeGlobals,
```

Comment:
shall we document this somehow?

Reply:
we probably should, but the events as of now are not documented so I'd rather do that separately at some point